Library/Homebrew/cask/config.rb

```diff
@@ -208,6 +210,16 @@ def languages=(languages)
       end
     end
 
+    sig { returns(T.any(String, Pathname)) }
+    def fontdir
+      get_dir_path(:fontdir)
```

Comment:
would be good if this or any of the other lines missing test coverage could have some added

Reply:
Yeah, this was actually part of my attempt to fix the `DEFAULTS` because the whole definition of methods based on hash keys made my head spin. 

If we want to keep the hash thing, I think that would work now as well. So I can remove these methods again if you prefer.

Reply:
Don't mind either way.